CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
 # Aries Cloud Agent Python Changelog
 
-## 0.12.7rc0
+## 0.12.7rc1
 
-### June 25, 2025
+### July 14, 2025
 
-This patch release adds a GitHub Action to publish an LTS container image with the tag `0.12-lts` when a release is published from the ACA-Py 0.12.lts branch. This is a convenience for those who want to use the latest LTS version of ACA-Py in their deployments, and is not intended to be used as a replacement for the latest release of ACA-Py or when a specific release is required. The `rc0` release candidate contains only the LTS release GitHub Actions change. We plan to cherry pick some additional PRs from the main branch for the final `0.12.7` release.
+This patch release adds a GitHub Action to publish an LTS container image with the tag `0.12-lts` when a release is published from the ACA-Py 0.12.lts branch. This is a convenience for those who want to use the latest LTS version of ACA-Py in their deployments, and is not intended to be used as a replacement for the latest release of ACA-Py or when a specific release is required.
+
+The release includes the following PRs from the `main` branch cherry-picked into this release:
+
+- Remove header from http/ws responses [\#3753](https://github.com/openwallet-foundation/acapy/pull/3753)
+- 🐛 Fix v2 cred ex and pres ex webhook events to emit after db write [\#3699](https://github.com/openwallet-foundation/acapy/pull/3699)
```

Comment:
https://github.com/openwallet-foundation/acapy/pull/3692 and https://github.com/openwallet-foundation/acapy/pull/3608 were backported for this.

Also I didn't actually backport https://github.com/openwallet-foundation/acapy/pull/3753 for this. I didn't think it was important enough to try and backport to the changed directory.